src/main/java/com/github/wovnio/wovnjava/HtmlConverter.java

```diff
@@ -18,6 +18,12 @@
     private final HashMap<String, String> hreflangMap;
     private final HtmlReplaceMarker htmlReplaceMarker;
 
+    private static final String[] WOVN_WIDGET_URLS = new String[] {
+        "j.wovn.io",
+        "j.dev-wovn.io:3000",
+        Settings.DefaultApiUrlBase
```

Comment:
I think this condition is enough.
But, WOVN.cs and Proxy have `/widget` to `https://wovn.global.ssl.fastly.net`.